items:
- uid: com.microsoft.azure.management.datalake.store.models._inner_error
  id: _inner_error
  parent: com.microsoft.azure.management.datalake.store.models
  children:
  - com.microsoft.azure.management.datalake.store.models._inner_error_1aee8baa98e9d57582c81432ea52e5d78d
  - com.microsoft.azure.management.datalake.store.models._inner_error_1af168200d5735757588d1a67b1ee26dcf
  href: com.microsoft.azure.management.datalake.store.models._inner_error.yml
  langs:
  - java
  name: InnerError
  fullName: com.microsoft.azure.management.datalake.store.models.InnerError
  type: Class
  source:
    remote: &o0
      path: azure-mgmt-datalake-store/src/main/java/com/microsoft/azure/management/datalake/store/models/InnerError.java
      branch: master
      repo: https://github.com/Azure/azure-sdk-for-java.git
    path: azure-mgmt-datalake-store/src/main/java/com/microsoft/azure/management/datalake/store/models/InnerError.java
    startLine: 15
  package: com.microsoft.azure.management.datalake.store.models
  summary: <p>Data Lake Store inner error information. </p>
  syntax:
    content: public class InnerError
  inheritance:
  - java.lang.Object
- uid: com.microsoft.azure.management.datalake.store.models._inner_error_1aee8baa98e9d57582c81432ea52e5d78d
  id: _inner_error_1aee8baa98e9d57582c81432ea52e5d78d
  parent: com.microsoft.azure.management.datalake.store.models._inner_error
  href: com.microsoft.azure.management.datalake.store.models._inner_error.yml
  langs:
  - java
  name: context()
  fullName: String com.microsoft.azure.management.datalake.store.models.InnerError.context()
  type: Method
  source:
    remote: *o0
    path: azure-mgmt-datalake-store/src/main/java/com/microsoft/azure/management/datalake/store/models/InnerError.java
    startLine: 42
  package: com.microsoft.azure.management.datalake.store.models
  summary: >-
    <p>Get the context value.</p>

    <p></p>
  syntax:
    content: public String context()
    return:
      type: "26831127"
      description: <p>the context value </p>
- uid: com.microsoft.azure.management.datalake.store.models._inner_error_1af168200d5735757588d1a67b1ee26dcf
  id: _inner_error_1af168200d5735757588d1a67b1ee26dcf
  parent: com.microsoft.azure.management.datalake.store.models._inner_error
  href: com.microsoft.azure.management.datalake.store.models._inner_error.yml
  langs:
  - java
  name: trace()
  fullName: String com.microsoft.azure.management.datalake.store.models.InnerError.trace()
  type: Method
  source:
    remote: *o0
    path: azure-mgmt-datalake-store/src/main/java/com/microsoft/azure/management/datalake/store/models/InnerError.java
    startLine: 33
  package: com.microsoft.azure.management.datalake.store.models
  summary: >-
    <p>Get the trace value.</p>

    <p></p>
  syntax:
    content: public String trace()
    return:
      type: "26831127"
      description: <p>the trace value </p>
references:
- uid: "26831127"
  spec.java:
  - name: String
    fullName: String
